src/main/java/core/basesyntax/dao/FruitOperationDaoImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitOperation;
+
+public class FruitOperationDaoImpl implements FruitOperationDao {
+    @Override
+    public void add(FruitOperation fruitOperation) {
+        Storage.SHOP_STORE.add(fruitOperation);
+    }
+
+    @Override
+    public FruitOperation get(String fruit) {
+        return Storage.SHOP_STORE.stream()
+                .filter(fruitOperation -> fruitOperation.getFruit().equals(fruit))
+                .findFirst()
+                .get();
```

Comment:
Using `get()` without `isPresent()` check may lead to NoSuchElementException if the element is not found. It's better to handle the absence of the element explicitly. Consider throwing a custom exception with a meaningful message if the fruit operation is not found.